common/src/main/java/com/frostwire/search/torrentz2/Torrentz2SearchResult.java

```diff
@@ -96,38 +96,43 @@ public String getTorrentUrl() {
     }
 
     private long parseCreationTime(String dateString) {
-        long result = 0;
+        long result = System.currentTimeMillis();
         try {
             String[] ds = dateString.split(" ");
-            if (ds[1].contains("hours")) {
+            if (ds[1].contains("hour")) {
```

Comment:
Awesome fixes

given they keep changing from lowercase to uppercase, perhaps it's best that we do this:
```java
// convert all the date string parts to lowercase
for (int i=0; i < ds.length; i ++) {
  ds[i] = ds[i].toLowerCase();
}
```

Then fix all the ifs below to ask for lower string counterparts

Reply:
> ds[1].toLowerCase()

would be enough it think

Reply:
I'm not sure if the java compiler will make an optimization for us, it'd be best if we can make sure those are converted to lower case only once.

Reply:
Also, Strings aren't mutable, if you do `s.toLowerCase()` it returns a new copy of the mutated string, but the original instance remains.